feat(app): GitHub integration (#576)
apify/apify-docs: sources/platform/integrations/github.md

### Create an Actor from a GitHub repository

> Learn how to create an Actor from a GitHub repository. This is useful if you want to use the GitHub integration to automatically deploy your code to Apify.

Reviewer: ```suggestion
Learn how to create an Actor from a GitHub repository. This is useful if you want to automatically deploy and build your code on push to your GitHub repository.
```

![New actor page](./images/apify-git-repository.png)

Selecting _Link Git repository_ will open a new modal to select a provider to use.
Selecting _GitHub_ will open a new window with GitHub authentication and selecting GitHub repository.

![Git provider select modal](./images/apify-git-provider-select.png)

> To link account, click on _Add GitHub account_ and follow instructions on github.com. Certain organization or user can be selected.

Reviewer 2: ```suggestion
To link an account, click on _Add GitHub account_ and follow the instructions on github.com. Certain organizations or users can be selected.
```

![Git repository account select](./images/apify-git-repository-add.png)

You can switch among all authorized users and organizations.

![Git repository account select](./images/apify-git-repository-account.png)

> If required repository is missing, try to find it with _Search_.

Reviewer 2: ```suggestion
If the required repository is missing, try finding it with _Search_.
```

![apify-git-repository-search.png](./images/apify-git-repository-search.png)

> Actor is created immediately from selected repository.

Reviewer 2: ```suggestion
An Actor is created immediately from the selected repository.
```
